src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -455,27 +461,32 @@ private void validateOutputSettings() {
     }
   }
 
-  // function: stringAlreadyInUseElsewhere
-  // purpose: Checks to make sure string isn't reserved or used by other fields
-  // param: string string to check
-  // param: field field name of provided string
-  private boolean stringAlreadyInUseElsewhere(String string, String field) {
-    boolean inUse = false;
+  private boolean stringConflictsWithReservedString(String string, String field) {
+    boolean reserved = false;
     for (String reservedString : TallyTransfers.RESERVED_STRINGS) {
       if (string.equalsIgnoreCase(reservedString)) {
-        inUse = true;
+        reserved = true;
         Logger.log(
             Level.SEVERE, "\"%s\" is a reserved term and can't be used for %s!", string, field);
         break;
       }
     }
+    return reserved;
+  }
+
+  // function: stringAlreadyInUseElsewhere
+  // purpose: Checks to make sure string isn't reserved or used by other fields
+  // param: string string to check
+  // param: field field name of provided string
+  private boolean stringAlreadyInUseElsewhereInSource(String string, CvrSource source, String field) {
```

Comment:
Might make sense to include this call as part of "basicCvrValidation" now since that would trigger it when you click "add" (if possible).

Reply:
ok